test/run/test_exports.py

```diff
@@ -0,0 +1,80 @@
+import contextlib
+import json
+import os
+import pathlib
+import shutil
+import tempfile
+from unittest.mock import patch
+
+import pytest
+
+from .. import test
+
+
+@pytest.fixture(name="jsondata", scope="module")
+def jsondata_fixture():
+    return json.dumps({
+        "version": "2",
+        "pipelines": [
+            {
+                "name": "image",
+                "stages": [
+                    {
+                        "type": "org.osbuild.truncate",
+                        "options": {
+                            "filename": "foo.img",
+                            "size": "10",
+                        },
+                    },
+                    {
+                        # cannot use org.osbuild.chown as it needs the chown
+                        # binary in the stage
+                        "type": "org.osbuild.testing.injectpy",
+                        "options": {
+                            "code": [
+                                'import os',
+                                'os.chown(f"{tree}/foo.img", 1000, 1000)',
+                            ],
+                        },
+                    },
+                ]
+            }
+        ]
+    })
+
+
+@pytest.fixture(name="osb", scope="module")
+def osbuild_fixture():
+    with test.OSBuild() as osb:
+        yield osb
+
+
+@pytest.fixture(name="injectpy", scope="module")
+def injectpy_fixture():
+    test_path = pathlib.Path(__file__).parent
+    inject_py_path = test_path / "../stages/org.osbuild.testing.injectpy"
+    stages_path = test_path / "../../stages/"
+    shutil.copy2(os.fspath(inject_py_path), os.fspath(stages_path))
```

Comment:
Another idea might be to copy all stages to a tmpdir and use the libdir flag.

Reply:
This needs to be done still!

Reply:
Thanks again for the suggestion and sorry that I forgot it earlier. This is done now, it's a little bit annoying to do because but it's cleaner this way.